### PR TITLE
Update Andrew-Harris.md

### PR DESCRIPTION
**Pull request by drwharris**

### CODE REVIEW

File: Andrew-Harris/Andrew-Harris.md [hunk 1]
```diff
@@ -25,16 +25,14 @@ quoteAuthor: "Leonardo Da Vinci"
 [[imgBadge]]
 | ![SQL Server](../badges/Developer-sql-server.png)
 
-Andrew is a **Senior Software Architect** with more than 25 years experience creating robust and performant solutions from the ground up. He has has worked in industries including Banking, Finance, Retail, People Management & eCommerce and has worked across platforms including Telephony, CRM, ERP, Payroll, Accounting and Banking.
+Andrew is a seasoned **Senior Software Architect** with over 25 years of experience modernisation and integrating Telephony, CRM, ERP, Payroll, Accounting, and Banking systems across sectors such as Banking, Finance, Retail, Hospitality, Sporting, People Management, and eCommerce.
```
Comment (drwharris) [inline Andrew-Harris/Andrew-Harris.md:28]:
```suggestion
Andrew is a seasoned **Senior Software Architect** with over 25 years of experience modernising and integrating Telephony, CRM, ERP, Payroll, Accounting, and Banking systems across sectors such as Banking, Finance, Retail, Hospitality, Sporting, People Management, and eCommerce. 

```

File: Andrew-Harris/Andrew-Harris.md [hunk 2]
```diff
@@ -25,16 +25,15 @@ quoteAuthor: "Leonardo Da Vinci"
 [[imgBadge]]
 | ![SQL Server](../badges/Developer-sql-server.png)
 
-Andrew is a **Senior Software Architect** with more than 25 years experience creating robust and performant solutions from the ground up. He has has worked in industries including Banking, Finance, Retail, People Management & eCommerce and has worked across platforms including Telephony, CRM, ERP, Payroll, Accounting and Banking.
+Andrew is a seasoned **Senior Software Architect** with over 25 years of experience modernising and integrating Telephony, CRM, ERP, Payroll, Accounting, and Banking systems across sectors such as Banking, Finance, Retail, Hospitality, Sporting, People Management, and eCommerce.
```
Comment (drwharris) [inline Andrew-Harris/Andrew-Harris.md:28]:
```suggestion
Andrew is a seasoned **Solution Architect** with over 25 years of experience modernising and integrating Telephony, CRM, ERP, Payroll, Accounting, and Banking systems across sectors such as Banking, Finance, Retail, Hospitality, Sporting, People Management, and eCommerce. 

```

**Review by Hona — APPROVED** [review]
Approved with suggestions

File: Andrew-Harris/Andrew-Harris.md [hunk 3]
```diff
@@ -25,16 +25,16 @@ quoteAuthor: "Leonardo Da Vinci"
...
+He is a passionate mentor combining his years of experience and his Scrum Master certification (CSM) to help guide developers through the intricacies of Scrum and the software development lifecycle. Blending technical and business acumen with a collaborative ethos, Andrew fosters environments where ideas flourish, and developers thrive, one line of code at a time.
+
+Andrew favourite tools and technologies include:
 
 * .NET Core
-* Angular
-* [Clean Architecture](https://rules.ssw.com.au/rules-to-better-clean-architecture)
+* Azure
 * Docker
 * Node.Js
```
Comment (Hona) [inline Andrew-Harris/Andrew-Harris.md:39]:
```suggestion
* Node.js
```

File: Andrew-Harris/Andrew-Harris.md [hunk 4]
```diff
@@ -25,16 +25,16 @@ quoteAuthor: "Leonardo Da Vinci"
...
+Andrew is a seasoned **Solution Architect** with over 25 years of experience modernising and integrating Telephony, CRM, ERP, Payroll, Accounting, and Banking systems across sectors such as Banking, Finance, Retail, Hospitality, Sporting, People Management, and eCommerce. 
 
-Andrew has a strong background in integration and application modernisation using technology including:
+
+
+He is a passionate mentor combining his years of experience and his Scrum Master certification (CSM) to help guide developers through the intricacies of Scrum and the software development lifecycle. Blending technical and business acumen with a collaborative ethos, Andrew fosters environments where ideas flourish, and developers thrive, one line of code at a time.
+
+Andrew favourite tools and technologies include:
 
 * .NET Core
```
Comment (Hona) [inline Andrew-Harris/Andrew-Harris.md:36]:
```suggestion
* .NET
```

**Review by Hona — APPROVED** [review]

**Pull request merged**